src/cdsetool/monitor.py

```diff
@@ -22,7 +22,7 @@
     from IPython import get_ipython  # type:ignore[reportMissingImports]
     from IPython.display import clear_output  # type:ignore[reportMissingImports]
 
-    if "IPKernelApp" not in get_ipython().config:
+    if "IPKernelApp" not in get_ipython().config:  # type:ignore
```

Comment:
It seems to run as expected, so my system might be bad somehow. 

I think i will add the python matrix to pyright also, just to ensure nothing is wrong on the supported versions

Reply:
I can't help you with that issue, unfortunately.

Something I've realized after making the initial PR is that pyright only reads `# type:ignore`, everything after that is ignored because of PEP-compliance. Mypy cares about the brackets and what is in them. Pyright has a pyright annotation that cares about the brackets though, so that is probably what I should have used.

Reply:
Thats good to know for the future :+1: 
For now i think the current type ignoring isnt critical, so they can wait until next iteration of those snippets